juju/model.py

```diff
@@ -1130,7 +1130,7 @@ def add_space(self, name, *cidrs):
 
         """
         key_facade = client.KeyManagerFacade.from_connection(self.connection())
-        return await key_facade.AddKeys([key], user)
+        return await key_facade.AddKeys(ssh_keys=[key], user=user)
```

Comment:
another where turning things to a list on passing in seems odd

Reply:
its another case where the shim is implementing "add_key()" singular, but the API is "add_keys()" plural